src/components/RangesDisplay.js

```diff
@@ -34,6 +36,23 @@ export const RangesDisplay = () => {
     return <Path key={id} id={id} d={drawArc(centerX, centerY, rangeRadius, start, end, true)} />;
   };
 
+  const _getAlignedRangesPath = (id, index) => {
+    const rangeRadius = radius + filledArcWidth;
+    const centerX = rangeRadius + margin;
+    const centerY = rangeRadius + margin;
+
+    const arc = arcs[index];
+    const previousArc = arcs[index - 1];
+
+    let end = 0;
```

Comment:
```suggestion
    let end = arcsStart + index * (arcSegmentDegree + spaceBetweenSegments);
```

Reply:
The end is already calculated in the SegmentedArc component, and duplicating the calculation here might result in deviations if we change it. The initial `end` should just be the start of the arc rather than 0, as arcs > 180 degrees have a negative degree starting point